exercises/practice/parallel-letter-frequency/.meta/example.rb

```diff
@@ -0,0 +1,23 @@
+class ParallelLetterFrequency
+  def self.count(texts)
+    ractors = (0...texts.length).map do |i|
+      Ractor.new(texts[i]) do |text|
```

Comment:
Is Ractor a gem that is not installed by default?

Reply:
Ractor may be a gem, but it is included in the core/standard libraries.